docs/v2-system-addon/data_events.md

```diff
@@ -1009,3 +1010,19 @@ This reports when an error has occurred when parsing/evaluating a JEXL targeting
   "value": ["MALFORMED_EXPRESSION" | "OTHER_ERROR"]
 }
 ```
+
+### Remote Settings error pings
+
+This reports a failure in the Remote Settings loader to load messages for Activity Stream Router.
+
+```js
+{
+  "action": "asrouter_undesired_event",
+  "client_id": "26288a14-5cc4-d14f-ae0a-bb01ef45be9c",
```

Comment:
Just to be clear, this ping also complies with the general telemetry policy for AS router. I.e: it uses `client_id` in the ping for all the pre-release channel and experiment audience in release. For regular release users, it reports `impression_id` other than `client_id`.